### ISSUE TITLE
Accessibility issues with Map control

### ISSUE THREAD

**Issue by mrlacey** [microsoft/microsoft-ui-xaml#2993]
**Describe the bug**

Some of the default behavior of the Map control causes it to fail basic accessibility testing.

**Steps to reproduce the bug**

1. Add map control to a page
2. Run app
3. Do not grant location permissions (not strictly necessary but not all issues will be seen in all locations)
4. Let map content fully load
5. Test the app with Accessibility Insights for Windows

**Expected behavior**

The default behavior of the control should not contain anything that causes it to fail basic accessibility checks.
Specifically [WCAG 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html) and [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology).

**Screenshots**

![image](https://user-images.githubusercontent.com/189547/88540811-90284f80-d00b-11ea-9a29-ab5e23ba9b7c.png)

**Version Info**

NuGet package version: Microsoft.UI.Xaml 2.4.2

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (20175)              |  Yes   |
| May 2020 Update (19041)            | Yes   |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            |  Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

This was discovered while reviewing the accessibility of applications generated by Windows Template Studio. (https://github.com/microsoft/WindowsTemplateStudio/issues/3723)

**Comment by StephenLPeters** [comment]
@MikeHillberg do you know what the plan for map control is?

**Comment by robloo** [comment]
> do you know what the plan for map control is?

This is also a show stopper for me switching to WinUI 3.0. This is a little off-topic but I'm curious when MapControl is going to be added in addition to when fixes like this get put in.

**Comment by YuliKl** [comment]
Based on information I received from the engineering team that owns MapControl, this behavior is by design, so I'm resolving this issue.

- Duplicate focusable sibling elements: the spatial nature of maps makes them difficult to represent faithfully to accessibility tools. We've taken the stance that all labels rendered on the map must appear in the UIA tree, and this means that elements with duplicate text can appear in the UIA tree.
- LocalizedControlType/name redundancy: testing with Narrator users in 2017 demonstrated that, unlike for other types of controls, it is helpful for users to hear "map" (or "map, {place}") as the first descriptor when they reach the MapControl. I'm hesitant to disrupt that user experience.
